lib/suspenders/app_builder.rb

```diff
@@ -499,8 +498,149 @@ def setup_default_rake_task
+      gem_description = 'Add gem for pretty view in browser & xls export for models?'
+      @@user_choice.push yes_no_question gem_name, gem_description
+    end
+
+    def faker_gem
+      gem_name = __callee__.to_s.gsub(/_gem/, '')
+      gem_description = 'Add gem for generate fake data in testing?'
+      @@user_choice.push yes_no_question gem_name, gem_description
+    end
+
+    def rubocop_gem
+      gem_name = __callee__.to_s.gsub(/_gem/, '')
+      gem_description = 'Add code inspector and code formatting tool?'
+      @@user_choice.push yes_no_question gem_name, gem_description
+    end
+
+    def guard_gem
+      gem_name = __callee__.to_s.gsub(/_gem/, '')
+      gem_description = 'Add guard (with livereliad) and dependences?'
+      @@user_choice.push yes_no_question gem_name, gem_description
+    end
+
+    def users_init_commit_choice
+      variants = { none: 'No', gitcommit: 'Make init commit at the end?' }
+      @@user_choice.push choice 'Commit? ', variants
+    end
+
+    def add_haml_gem
+      inject_into_file('Gemfile', "\ngem 'haml-rails'", after: '# user_choice')
+    end
+
+    def add_slim_gem
+      inject_into_file('Gemfile', "\ngem 'slim-rails'", after: '# user_choice')
+    end
+
+    def add_rails_db_gem
+      inject_into_file('Gemfile', "\n  gem 'rails_db'\n  gem 'axlsx_rails'", after: 'group :development do')
+    end
+
+    def add_rubocop_gem
+      inject_into_file('Gemfile', "\n  gem 'rubocop', require: false", after: 'group :development do')
+    end
+
+    def add_guard_gem
+      t=<<-TEXT.chomp
+
+  gem 'guard'
+  gem 'guard-livereload', '~> 2.4', require: false
+      TEXT
+      inject_into_file('Gemfile', t, after: 'group :development do')
+    end
+
+    def add_guard_rubocop_gem
+      inject_into_file('Gemfile', "\n  gem 'guard-rubocop'", after: 'group :development do')
+    end
+
+    def add_meta_request_gem
+      inject_into_file('Gemfile', "\n  gem 'meta_request'", after: 'group :development do')
+    end
+
+    def add_faker_gem
+      inject_into_file('Gemfile', "\n  gem 'faker'", after: 'group :test do')
+    end
+
+    def add_user_gems
+      add_haml_gem           if @@user_choice.include? :haml
+      add_slim_gem           if @@user_choice.include? :slim
+      add_rails_db_gem       if @@user_choice.include? :rails_db
+      add_faker_gem          if @@user_choice.include? :faker
+      add_meta_request_gem   if @@user_choice.include? :meta_request
+      add_rubocop_gem        if @@user_choice.include? :rubocop
+      add_guard_gem          if @@user_choice.include? :guard
+      add_guard_rubocop_gem  if @@user_choice.include?(:guard) && @@user_choice.include?(:rubocop)
+    end
+
+    def post_init
+      run 'guard init' if @@user_choice.present? && @@user_choice.include?(:guard)
+      if @@user_choice.include? :rubocop
+        t=<<-TEXT
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new
+        TEXT
+        append_file 'Rakefile', t
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.